util/execution/table_generator/table_generator.cpp

```diff
@@ -14,14 +14,13 @@
 
 namespace terrier::execution::sql {
 template <typename T>
-T *TableGenerator::CreateNumberColumnData(Dist dist, uint32_t num_vals, uint64_t min, uint64_t max) {
-  static uint64_t serial_counter = 0;
+T *TableGenerator::CreateNumberColumnData(ColumnInsertMeta *col_meta, uint32_t num_vals) {
```

Comment:
I don't see any functional change here. Any other reason to include in this PR?

Reply:
Per Amadou: needed to get rid of the `static` to be able to reuse it multiple times in test cases.